### PR TITLE
Change Radar orientation to FLU

### PR DESCRIPTION
**Pull request by lukicdarkoo**
**Documentation**
https://cyberbotics.com/doc/reference/radar?version=a708be1b9#description

### CODE REVIEW

File: src/webots/nodes/utils/WbObjectDetection.cpp [hunk 1]
```diff
@@ -368,19 +368,19 @@ WbAffinePlane *WbObjectDetection::computeFrustumPlanes(const WbVector3 &devicePo
...
-  double x = maxRange * tan(horizontalFieldOfView / 2.0);
-  double z = -maxRange;
-  double y = maxRange * tan(verticalFieldOfView / 2.0);
-  const WbVector3 topRightCorner = devicePosition + deviceRotation * WbVector3(x, y, z);
-  const WbVector3 topLeftCorner = devicePosition + deviceRotation * WbVector3(-x, y, z);
-  const WbVector3 bottomRightCorner = devicePosition + deviceRotation * WbVector3(x, -y, z);
-  const WbVector3 bottomLeftCorner = devicePosition + deviceRotation * WbVector3(-x, -y, z);
+  double z = maxRange * tan(verticalFieldOfView / 2.0);
+  double y = maxRange * tan(horizontalFieldOfView / 2.0);
+  double x = maxRange;
```
Comment (omichel) [inline src/webots/nodes/utils/WbObjectDetection.cpp:373]: Can't we make these const?:
```suggestion
  const double z = maxRange * tan(verticalFieldOfView / 2.0);
  const double y = maxRange * tan(horizontalFieldOfView / 2.0);
  const double x = maxRange;
```

File: src/webots/nodes/utils/WbObjectDetection.cpp [hunk 2]
```diff
@@ -308,7 +308,7 @@ bool WbObjectDetection::computeBounds(const WbVector3 &devicePosition, const WbM
           assert(false);
       }
       if (nodeType == WB_NODE_CYLINDER || nodeType == WB_NODE_CAPSULE) {
-        WbMatrix3 rotation = deviceInverseRotation * objectRotation;
+        WbMatrix3 rotation = deviceRotation * objectRotation;
         double xRange = fabs(rotation(0, 1) * height) + 2 * radius * sqrt(qMax(0.0, 1.0 - rotation(0, 1) * rotation(0, 1)));
         double yRange = fabs(rotation(1, 1) * height) + 2 * radius * sqrt(qMax(0.0, 1.0 - rotation(1, 1) * rotation(1, 1)));
         double zRange = fabs(rotation(2, 1) * height) + 2 * radius * sqrt(qMax(0.0, 1.0 - rotation(2, 1) * rotation(2, 1)));
```
Comment (omichel) [inline src/webots/nodes/utils/WbObjectDetection.cpp:314]: Maybe some const here:
```suggestion
        const WbMatrix3 rotation = deviceRotation * objectRotation;
        const double xRange = fabs(rotation(0, 1) * height) + 2 * radius * sqrt(qMax(0.0, 1.0 - rotation(0, 1) * rotation(0, 1)));
        const double yRange = fabs(rotation(1, 1) * height) + 2 * radius * sqrt(qMax(0.0, 1.0 - rotation(1, 1) * rotation(1, 1)));
        const double zRange = fabs(rotation(2, 1) * height) + 2 * radius * sqrt(qMax(0.0, 1.0 - rotation(2, 1) * rotation(2, 1)));
```

File: src/webots/nodes/utils/WbObjectDetection.hpp [hunk 3]
```diff
@@ -46,6 +46,9 @@ class WbObjectDetection {
   bool recomputeRayDirection(WbSolid *device, const WbVector3 &devicePosition, const WbMatrix3 &deviceRotation,
                              const WbMatrix3 &deviceInverseRotation, const WbAffinePlane *frustumPlanes);
 
+  // Checks whether the object is detected.
+  // \n\n
```
Comment (omichel) [inline src/webots/nodes/utils/WbObjectDetection.hpp:50]: This `\n\n` comment is odd.

File: src/webots/nodes/utils/WbObjectDetection.hpp [hunk 4]
```diff
@@ -46,6 +46,9 @@ class WbObjectDetection {
...
+  // TODO: Check whether the parameters are needed as they are already provided in the class constructor.
```
Comment (omichel) [inline src/webots/nodes/utils/WbObjectDetection.hpp:51]: Can we address this TODO in this PR?

File: src/webots/nodes/utils/WbObjectDetection.hpp [hunk 5]
```diff
@@ -60,6 +63,23 @@ class WbObjectDetection {
   bool recursivelyComputeBounds(WbSolid *solid, bool boundsInitialized, const WbVector3 &devicePosition,
                                 const WbMatrix3 &deviceRotation, const WbMatrix3 &deviceInverseRotation,
                                 const WbAffinePlane *frustumPlanes);
+
+  // Checks whether the object is in the bounds of the `frustumPlanes` frustum.
+  // \n\n
```
Comment (omichel) [inline src/webots/nodes/utils/WbObjectDetection.hpp:68]: This `\n\n` comment looks odd.

**Review by omichel — CHANGES_REQUESTED** [review]

File: src/webots/nodes/utils/WbObjectDetection.hpp (hunk 4 above)
Comment (omichel) [reply src/webots/nodes/utils/WbObjectDetection.hpp:51]: Yes, opening an issue would be nice otherwise I fear it will never be addressed.

**Review by omichel — APPROVED** [review]
Thank you.

File: src/webots/nodes/WbCamera.cpp [hunk 6]
```diff
@@ -848,6 +848,7 @@ void WbCamera::createWrenCamera() {
   } else {
     mSegmentationCamera = NULL;
     disconnect(mSensor, &WbSensor::stateChanged, this, &WbCamera::updateOverlayMaskTexture);
+    disconnect(mSegmentationCamera, &WbWrenCamera::cameraInitialized, this, &WbCamera::updateSegmentationCameraOrientation);
```
Comment (omichel) [inline src/webots/nodes/WbCamera.cpp:851]: We are setting `mSegmentationCamera` to NULL and then we are disconnecting it (NULL).

Comment (lukicdarkoo) [reply src/webots/nodes/WbCamera.cpp:851]: I am not sure why I attached that event, maybe it slipped during some tests.

**Pull request merged**